dbt/task/compile.py

```diff
@@ -31,6 +59,15 @@ def __project_sources(self, project):
                     if fnmatch.fnmatch(filename, "*.sql"):
                         indexed_files[full_source_path].append(rel_path)
 
+                        # TODO : include project name here!
```

Comment:
this is great.  if you want to just add the project name functionality into this branch, go for it.